spark/src/main/java/org/apache/iceberg/spark/TypeToSparkType.java

```diff
@@ -104,12 +104,14 @@ public DataType primitive(Type.PrimitiveType primitive) {
         throw new UnsupportedOperationException(
             "Spark does not support time fields");
       case TIMESTAMP:
-        Types.TimestampType timestamp = (Types.TimestampType) primitive;
-        if (timestamp.shouldAdjustToUTC()) {
-          return TimestampType$.MODULE$;
-        }
-        throw new UnsupportedOperationException(
-            "Spark does not support timestamp without time zone fields");
+//        Types.TimestampType timestamp = (Types.TimestampType) primitive;
```

Comment:
Need to remove these comment and this is where I think this is where we have the Precondtion.checkArguement(HandleTimeStampWithoutTimezone, Message)